src/ontology/mondo-ingest.Makefile

```diff
@@ -413,18 +413,20 @@ tmp/merged.owl: tmp/mondo.owl mondo-ingest.owl tmp/mondo.sssom.ttl
 	$(ROBOT) merge -i tmp/mondo.owl -i mondo-ingest.owl -i tmp/mondo.sssom.ttl --add-prefixes config/context.json \
 			 remove --term "http://purl.obolibrary.org/obo/mondo#ABBREVIATION" --preserve-structure false -o $@
 
-tmp/merged.db: tmp/merged.owl
+%.db: %.owl
```

Comment:
## Add: `%.db`
We previously had a lot of duplicate code (literally exact same goal recipe/commands) for the creation of these `.db` files. It seems overdue that this be captured with a wildcard.

This will also properly handle `tmp/*.db` cases, not just `*.db`.